can_spec_my18.yml

```diff
@@ -681,6 +671,19 @@ buses:
             position: 60
             length: 1
             c_type: bool
+      DashRequestLC:
+        can_id: 0x0D5
+        is_big_endian: true
+        period: 69ms
+        segments:
+          using_launch_ctrl:
+            position: 0
+            length: 1
+            c_type: bool
+          launch_ctrl_slip_ratio: # Ranges from 105 to 115
+            position: 16
+            length: 16
+            c_type: uint16_t
```

Comment:
Could we use a uint8_t instead since it only ranges between 105 and 115?

Reply:
Premature opt.